ledger-core/.clang-tidy

```diff
@@ -0,0 +1 @@
+../.clang-tidy
```

Comment:
Do we have to add the symlink here ? A root .clang-tidy should be enough isn't it ?